js/controller.js

```diff
@@ -1603,12 +1603,18 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     setClosedCaptionsLanguage: function(){
       var availableLanguages = this.state.closedCaptionOptions.availableLanguages;
-      //if saved language not in available languages, set to first available language
-      if (availableLanguages && (this.state.closedCaptionOptions.language == null || !_.contains(availableLanguages.languages, this.state.closedCaptionOptions.language))) {
+
+      //if saved language not in available languages and saved is not 'none', set to first available language
+      if (availableLanguages && this.state.closedCaptionOptions.language !== CONSTANTS.CLOSED_CAPTIONS.NONE_LANGUAGE
+        && (this.state.closedCaptionOptions.language == null || !_.contains(availableLanguages.languages, this.state.closedCaptionOptions.language))) {
```

Comment:
Would recommend using a separate line for each condition since this `if` statement is now a bit long. We usually leave the operator to the right, such as
```
if (availableLanguages && 
    this.state.closedCaptionOptions.language !== CONSTANTS.CLOSED_CAPTIONS.NONE_LANGUAGE && 
    (this.state.closedCaptionOptions.language == null || 
    !_.contains(availableLanguages.languages, this.state.closedCaptionOptions.language))) {
```
Looks good otherwise

Reply:
Done.